src/cplus_plugin/gui/constant_raster_manager_dialog.py

```diff
@@ -314,7 +325,9 @@ def _initialize(self):
 
         if YEARS_EXPERIENCE_ACTIVITY_ID in metadata_ids:
```

Comment:
@osundwajeff alll variables such as `YEARS_EXPERIENCE_ACTIVITY_ID` should be defined in the definitions modules either in `defaults.py` or `constants.py`.